framework/infrastructure/gcp/compute.py

```diff
@@ -558,6 +567,60 @@ def get_backend_service_backend_health(self, backend_service, backend):
             .execute()
         )
 
+    def create_serverless_neg(self, name: str, region: str, service_name: str):
+        """Creates a serverless NEG.
+
+        Args:
+            name: The name of the NEG.
+            region: The region in which to create the NEG.
+            service_name: The name of the Cloud Run service.
+            service_name format is "namespaces/{namespace}/services/{service}"
+
+        Returns:
+            The NEG selfLink URL
+        """
+
+        neg_body = {
+            "name": name,
+            "networkEndpointType": "SERVERLESS",
+            # "Note: Cloud Run is not the only service that uses serverless NEGs
+            # but we are only supporting Cloud Run right now."
+            "cloudRun": {"service": service_name},
+        }
+        logger.info("Creating serverless NEG %s in %s", name, region)
+        self._insert_resource(
+            self.api.regionNetworkEndpointGroups(),
+            neg_body,
+            region=region,
+        )
+
+        neg = self.get_serverless_network_endpoint_group(name, region)
+        logger.info("Created serverless neg %s ", neg)
```

Comment:
be consistent in spelling, elsewhere it's called NEG
```suggestion
        logger.info("Created serverless NEG %s ", neg)
```